containers/kubernetes/api-cli/cluster-monitoring.mdx

```diff
@@ -162,8 +162,8 @@ The `loki` application is not included in the default Helm repositories.
                                     --namespace loki-stack \
                                     --set promtail.enabled=true,loki.persistence.enabled=true,loki.persistence.size=100Gi
     ```
-    It will use the Kapsule's default storage class which is `scw-bsdd` to create block volumes using Scaleway Block Storage.
-    If you plan to use Loki on a production system be sure that you set up a retention period to avoid filling the filesystems. For instance use these parameters if you want to enable a 30 days retention (logs older than 30 days will be deleted). Note that you have to choose the size of the persistent volume to fit the amount of volume your deployment will create.
+    It will use Kapsule's default storage class, `scw-bsdd`, to create block volumes using Scaleway Block Storage.
+    If you plan to use Loki on a production system, ensure to set up a retention period to avoid filling the file systems. For instance, use these parameters if you want to enable a 30-day retention (logs older than 30 days will be deleted). Note that you have to choose the size of the persistent volume to fit the amount of volume your deployment will create.
```

Comment:
```suggestion
    If you plan to use Loki on a production system, ensure that you set up a retention period to avoid filling the file systems. For instance, use these parameters if you want to enable a 30-day retention (logs older than 30 days will be deleted). Note that you have to choose the size of the persistent volume to fit the amount of volume your deployment will create.
```